index.js

```diff
@@ -34,7 +34,7 @@ function MapboxLanguage(options) {
     }
   };
   this._excludedLayerIds = options.excludedLayerIds || [];
-  this.supportedLanguages = options.supportedLanguages || ['en', 'es', 'fr', 'de', 'ru', 'zh', 'ar', 'pt'];
+  this.supportedLanguages = options.supportedLanguages || ['local', 'en', 'es', 'fr', 'de', 'ru', 'zh', 'ar', 'pt'];
```

Comment:
I think it would be better to use `mul` (the ISO 639 code for “multilingual”) to represent this option, as the iOS and macOS map SDKs do ([documentation](https://www.mapbox.com/ios-sdk/api/4.1.1/Classes/MGLStyle.html#/c:objc%28cs%29MGLStyle%28im%29localizeLabelsIntoLocale:)). That way, all the `supportedLanguages` values are valid IETF language tags, which will avoid edge cases when implementing #5.

Reply:
I fully agree with you.
Fixed it.

Thanks!